### ISSUE TITLE
How to return answer instead of just printing it?

### ISSUE THREAD

**Issue by Tylersuard** [microsoft/autogen#784]
When I run an agent, the agent's thought process and answer gets printed out to the terminal.  However, I want to call the agent from a different function and have the agent return its answer to the function.  How should I do that?

**Comment by victordibia** [comment]
@Tylersuard .. does this address your issues/.

**Comment by Tylersuard** [comment]
@victordibia   Very helpful, thank you!

**Comment by sagearbor** [comment]
@victordibia  clearly user error on my part but I could only get the manager in my multichat to output their last message.  I want all the other agents dialog to get logged to.  For example at the end of my script I have the first two lines below (many agents) , then I initiate chat (line 3), if I include line 4 I just get the prompt message I gave to my manager written to output.txt, and if I include line5 my output.txt is empty even though "mdDoc" had output I saw in the terminal.

groupchat = autogen.GroupChat(agents=[mdDoc,doDoc,docHarmGen,docHarmDose], messages=[], max_round=3)

manager = autogen.GroupChatManager(groupchat=groupchat, llm_config=llm_config)

user_proxy.initiate_chat(manager, message=task)

json.dump(user_proxy.chat_messages[manager], open(output.txt, "w"), indent=2)

json.dump(user_proxy.chat_messages[mdDoc], open(output.txt, "w"), indent=2)
